Jenkinsfile

```diff
@@ -0,0 +1,497 @@
+                        env.playbook = "ci-pipeline/config/duffy-setup/setup-rpmbuild-system.yml"
+                        env.ref = "fedora/${fed_branch}/${basearch}/atomic-host"
+                        env.repo = "${env.fed_repo}"
+                        env.rev = "${env.fed_rev}"
+                        env.basearch = "x86_64"
+                        env.ANSIBLE_HOST_KEY_CHECKING = "False"
+                        env.DUFFY_OP = "--allocate"
+
+                        // Send message on fedmsg
+                        sendCIMessage messageContent: '',
+                                messageProperties: "topic=${topic}\n" +
+                                        "build_url=${BUILD_URL}\n" +
+                                        "build_id=${BUILD_ID}\n" +
+                                        "branch=${branch}\n" +
+                                        "ref=fedora/${branch}/${basearch}/atomic-host\n" +
+                                        "rev=${fed_rev}\n" +
+                                        "repo=${fed_repo}\n" +
+                                        "namespace=${namespace}\n" +
+                                        "username=fedora-atomic\n" +
+                                        "test_guidance=''\n" +
+                                        "status=success",
+
+                                messageType: 'Custom',
+                                overrides: [topic: "${topic}"],
+                                providerName: 'fedora-fedmsg'
+
+                        // Provision of resources
+                        allocDuffy("${current_stage}")
+
+                        echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
+                                "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
+                                "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
+                                "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
+                        def job_props = "${env.ORIGIN_WORKSPACE}/job.props"
+                        def job_props_groovy = "${env.ORIGIN_WORKSPACE}/job.groovy"
+                        convertProps(job_props, job_props_groovy)
+                        load(job_props_groovy)
+
+                        // Stage resources - RPM build system
+                        setupStage()
+
+                        // Rsync Data
+                        writeFile file: "${env.ORIGIN_WORKSPACE}/task.env",
+                                  text: "export JENKINS_JOB_NAME=\"${JOB_NAME}-${current_stage}\"\n" +
+                                        "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
+                                        "export OSTREE_BRANCH=\"\${OSTREE_BRANCH:-}\"\n" +
+                                        "export fed_repo=\"${fed_repo}\"\n" +
+                                        "export fed_branch=\"${fed_branch}\"\n" +
```

Comment:
In line 66 we changed fed_branch to rawhide if it was master.  Is it still rawhide here?  Not sure about variables in the different stages.  If it is still master and the variable change persists, we need to not change fed_branch in line 66.  The reason for this is that the rpmbuild Dockerfile entry point relies on fed_branch being master for the rawhide case, as it does a git checkout ${fed_branch}